openpype/hosts/houdini/plugins/publish/validate_workfile_paths.py

```diff
@@ -24,6 +24,16 @@ class ValidateWorkfilePaths(
     node_types = ["file", "alembic"]
     prohibited_vars = ["$HIP", "$JOB"]
 
+    @classmethod
+    def apply_settings(cls, project_settings, system_settings):
+        """Apply project settings to creator"""
+        settings = (
+            project_settings["houdini"]["publish"]["ValidateWorkfilePaths"]
+        )
+
+        cls.node_types = settings.get("node_types", cls.node_types)
+        cls.prohibited_vars = settings.get("prohibited_vars", cls.prohibited_vars)
+
```

Comment:
That is default behavior of automated settings application.
```suggestion
```

Reply:
So, is this automation happening for vslidators only ? 

---

I found that automation is used in some maya validators. 

However, it's explicitly done in this validator

https://github.com/ynput/OpenPype/blob/develop/openpype/hosts/maya/plugins/publish/validate_maya_units.py#L39-L52

Reply:
The automated behavior is described here](https://openpype.io/docs/dev_publishing#plugin-settings). Not sure if the behavior is also for Creators. :)

Reply:
It is not for Creators.

Reply:
Oh, thanks for the link. I will check it.

> Not sure if the behavior is also for Creators.

I kept looking in creators super classes and found there is empty `apply_settings` in creator